### ISSUE TITLE
Mycroft playing on Soundbar not on itself Mycroft

### ISSUE THREAD

**Issue by VilterPD** [forslund/spotify-skill#144]
Mycroft playing on my Yamaha YAS-109 Soundbar intstead of on mycroft.
Mycroft does not show up on Spotify Connect either.

Thanks for all your great work

**Comment by VilterPD** [comment]
just realzed raspspotify was missing

**Comment by forslund** [comment]
Yes a separate spotify player is required since the skill basically just handles the Spotify connect protocol.

Make sure you edit the raspotify configuration and
- set the device name to the same as you've named the Mycroft device when you registered it on home.mycroft.ai.
- set the username and password (if these are missing raspotify is in discover mode and the skill can't see it until another device logs in on it)

**Comment by forslund** [comment]
Hi, did you get this working or is it still an issue?
